lib/broadway.ex

```diff
@@ -163,10 +162,10 @@ defmodule Broadway do
 
   The configuration above defines a pipeline with:
 
-    * 1 producer
-    * 2 processors
-    * 1 batcher named `:sqs` with 2 batch processors
-    * 1 batcher named `:s3` with 1 batch processors
+    * One producer
+    * Two processors
+    * One batcher named `:sqs` with two batch processors
+    * One batcher named `:s3` with one batch processor
```

Comment:
I ran an internal survey about this change and so far it looks like most people here prefer keeping the numbers. I wonder if you have an argument that can change their minds :)

Reply:
The normal rule is to spell out all numbers less than 10. 
https://www.grammarly.com/blog/when-to-spell-out-numbers/
https://blog.apastyle.org/apastyle/2014/06/comparing-mla-and-apa-numbers.html

Reply:
The theory is that when we read we actually sound out the numbers in our heads. So you see rules like "if it is less than two syllables, spell it out". That's not necessarily true for non native speakers, and programmers in general. For them, numbers may be easier. 
It still looks a bit weird/amateurish to have bare numbers, so I think it's better to follow the normal technical writing rules for documentation.

Reply:
I guess I am an amateur then, numbers are definitely easier to read for me. :D

Reply:
Well, I agree that the numbers are easier to read. It's just that this is one of the standard rules that any professional editor would follow, and not doing it makes the text look different. Many of these rules are arbitrary, but going along makes it easier to work with editors, and we can save the fights for the things that matter.